tests/e2e/util.go

```diff
@@ -3754,6 +3867,72 @@ func waitForCNSRegisterVolumeToGetDeleted(ctx context.Context, restConfig *rest.
 	return fmt.Errorf("CnsRegisterVolume %s deletion is failed within %v", cnsRegisterVolumeName, timeout)
 }
 
+func execCommanOnWindowsWorker(ctx context.Context, client clientset.Interface, windowsWorkerIP string) int64 {
+	//ips := getK8sMasterIPs(ctx, client)
+	//k8sMasterIP := ips[0]
+	windowsUser := os.Getenv("WINDOWS_USER")
+	jumpVMIP := os.Getenv("JUMP_VM_IP")
```

Comment:
Please check and re write this function code. the code is duplicated from the other function